scripts/download_embeddings_and_vector_db.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This scripts run outside the container, just in the local host? I am a bit lost on these changes @TamiTakamiya , maybe let's talk tomorrow and you can explain us what's the idea behind please